### ISSUE TITLE
Execution failed for task ':flutter_aepedgebridge:packageDebugResources'.

### ISSUE THREAD

**Issue by amitaman** [adobe/aepsdk_flutter#89]
Hi Team,
I was trying to update my Gradle to 8.2.2 using AGP assistant and getting this error for AEP packages. I have a namespace variable in build.gradle file. 

```
android {
    namespace 'com.example.xxxx'
    compileSdkVersion 34
    ndkVersion flutter.ndkVersion

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

    kotlinOptions {
        jvmTarget = '1.8'
    }
}
```

### Expected Behaviour
The build should pass without any error.

### Actual Behaviour
Build is failing with the below error in **Android Studio:**
Namespace not specified. Please specify a namespace in the module's build.gradle file like so:

android {
    namespace 'com.example.namespace'
}

If the package attribute is specified in the source AndroidManifest.xml, it can be migrated automatically to the namespace value in the build.gradle file using the AGP Upgrade Assistant; please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.

**In VS Code:**
         > Package Name not found in /Users/amiaman/.pub-cache/hosted/pub.dev/flutter_aepedgebridge-3.0.0/android/src/main/AndroidManifest.xml, and namespace not specified. Please specify a namespace for the generated R and BuildConfig classes via android.namespace in the module's build.gradle file like so:

           android {
               namespace 'com.example.namespace'
           }
3

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4s
Exception: Gradle task assembleDebug failed with exit code 1

### Reproduce Scenario (including but not limited to)
Try to update the build.gradle from 7.4.2 to 8.x.x

#### Platform and Version
Platform - Android
Version - API 32

I believe the package name should be moved to namespace field in build.gadle file for the pakages.

**Comment by ahmyr** [comment]
@amitaman, we don't yet support Gradle 8 or namespaces. It is on our backlog, but no ETA yet. The project is open source though, so feel free to make the needed changes yourself.

**Comment by amitaman** [comment]
Hi @ahmyr thanks for the reply. Can you please confirm what is the latest Gradle support is there for AEP. Also want to know is the AEP packages having a support of iOS 17.

**Comment by cacheung** [comment]
Hi @amitaman the namespace support for Android has been released with aepsdk_flutter plugins[ version 4.0.1](https://github.com/adobe/aepsdk_flutter/releases).
It should resolve this error.

**Comment by cacheung** [comment]
Closing this ticket, please log a new one if you have more questions.
